### PR TITLE
Add voltage regulators to Congo and Morocco dts

### PR DESCRIPTION
**Pull request by abinayaddhandapani**
ARM64: dts: aspeed: Add voltage regulators to Congo and Morocco dts files

### CODE REVIEW

**Comment by ojayanth** [comment]
@abinayaddhandapani  please follow the commit message with same format follwed other dts commits . refer any of the changes specific to dts file for refernce.

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts [hunk 1]
```diff
@@ -64,8 +64,12 @@
 	aliases {
 
 #ifdef EEPROM_PROG_ENABLE
-		i2c100 = &channel_0_0;
+		i2c100 = &p0_channel_0_0;
+		i2c200 = &p1_channel_0_0;
 #endif
+		i2c102 = &p0_channel_0_2;
+		i2c202 = &p1_channel_0_2;
```
Comment (mahkurap) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts]: to maintain the bus numbers in ascending order, move i2c202 after i2c123.

**Review by mahkurap — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts (hunk 1 above)
Comment (abinayaddhandapani) [reply arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts]: Ack

**Comment by abinayaddhandapani** [comment]
> @abinayaddhandapani please follow the commit message with same format follwed other dts commits . refer any of the changes specific to dts file for refernce.

Ack

**Review by supven01 — APPROVED** [review]

**Review by mahkurap — APPROVED** [review]

**Pull request merged**